napari/utils/_tests/test_geometry.py

```diff
@@ -78,6 +80,22 @@ def test_intersect_line_with_plane_3d(
     np.testing.assert_allclose(expected, intersection)
 
 
+def test_intersect_line_with_multiple_planes_3d():
+    """Test intersecting a ray with multiple planes and getting the intersection
+    with each one.
+    """
```

Comment:
we should refactor so that the only func we have for this is a vectorised line-multiple plane intersection which returns the squeezed intersection point if only 1 is returned - again, not blocking for this, just noting things 🙂